server/models/index.js

```diff
@@ -14,21 +14,16 @@ module.exports = function () {
   const app = this;
 
   let sequelize;
-  if (process.env.NODE_ENV === 'production') {
-    sequelize = sequelizeHeroku.connect(Sequelize);
```

Comment:
I heard that you dropped Heroku so this change should be ok? Maybe this was why the `NODE_ENV` wasn't production on the production commands :smile: